src/components/Atoms/Alert/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import BREAKPOINT from "../../../variables/breakpoint";
+
+const Alert = ({ visible, errorText }) => {
+  return (
+    <AlertWrapper isActive={visible}>
+      <AlertText> {errorText} </AlertText>
+    </AlertWrapper>
+  );
+};
+
+const AlertText = styled.div`
+  color: ${COLOR.WHITE};
+  font-size: ${TEXT.S};
+  width: 100%;
+  flex-basis: 400px;
+  padding: 10px 20px;
+  background-color: ${COLOR.RED};
+  border-radius: 4px;
+`;
+
+const AlertWrapper = styled.div`
+  width: 100%;
+  display: flex;
```

Comment:
```suggestion
const AlertWrapper = styled.div`
  z-index: 1000;
  width: 100%;
  display: flex;
```
こうすると、ボタンのテキストがAlertより手前に来ている問題を解決できます。